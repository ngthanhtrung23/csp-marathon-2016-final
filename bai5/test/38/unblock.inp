Z.22T.
ZDDDT.
00567.
RR567.
...9AA
...9..
