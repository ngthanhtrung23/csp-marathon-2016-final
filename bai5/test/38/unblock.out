34
2 L 1
5 D 2
0 R 1
R R 1
Z D 4
0 L 1
D L 1
6 U 2
9 U 2
R L 1
5 U 2
A L 3
7 D 2
9 D 2
6 D 2
T D 2
2 R 2
D R 3
5 U 2
0 R 1
R R 1
Z U 4
0 L 1
A L 1
R L 1
5 D 4
0 R 1
2 L 2
R R 1
Z D 2
D L 3
6 U 2
T U 2
0 R 3

